app/views/documents/show/_history_tab.html.erb

```diff
@@ -24,7 +24,7 @@
           <%= simple_format(escape_and_link(entry.details.body)) %>
         <% end %>
 
-        <% if entry.withdrawn? && entry.details %>
+        <% if (entry.withdrawn? || entry.withdrawn_updated?) && entry.details %>
```

Comment:
Is `&& entry.details` necessary here as if the document has been withdrawn the `details` for a TimelineEntry would have been set to the withdrawal?

Reply:
As per the UnpublishService

Reply:
It's not essential it's just defensive. I guess it depends on how deeply we want to guarantee that a details object exists for certain TimelineEntry types. 

It's consistent with all the other timeline entry though, we could indeed remove all of them as our current doesn't present opportunities that they could be a miss - but on the other side we have nothing to guarantee their presence.

Reply:
Hmm yep i see what you mean, however I'd be in favour of potentially removing them as from the code you couldn't create an entry without associating it with a `detail` as an exception would be thrown and the entry would never of been created in the first place?

This is maybe something out of scope for this pr though

Reply:
Yeah no problems if you want to remove it in a separate PR.

I personally tend to veer towards defensive code - though I don't think I wrote it originally - so it doesn't feel too concerning to me but no qualms with it leaving either.